source/security/authentication.txt

```diff
@@ -0,0 +1,390 @@
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-256-client
+         :end-before: end-scram-sha-256-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-scram-sha-256-uri
+         :end-before: end-scram-sha-256-uri
+
+.. _php-scram-sha-1:
+
+SCRAM-SHA-1
+-----------
+
+SCRAM-SHA-1, as defined by `RFC 5802 <https://www.rfc-editor.org/rfc/rfc5802>`__,
+is the default authentication mechanism on MongoDB deployments
+running {+mdb-server+} v3.6.
+
+.. note::
+
+   {+php-library+} v1.20 drops support for {+mdb-server+} v3.6. When using v1.20+ of
+   the library, all supported server versions default to the SCRAM-SHA-256 authentication
+   mechanism.
+
+To authenticate with this mechanism, use the same syntax as the :ref:`php-scram-sha-256`,
+but change the value of the ``authMechanism`` option to ``'SCRAM-SHA-1'``.
+
+.. _php-mongodb-x509:
+
+MONGODB-X509
+------------
+
+If you enable TLS, during the TLS handshake, the {+php-library+} can present an X.509
+client certificate to MongoDB to prove its identity. The MONGODB-X509 authentication
+mechanism uses this certificate to authenticate the client.
+
+To authenticate with this mechanism, set the following connection options:
+
+- ``tls``: Set to ``true``.
+- ``tlsCertificateKeyFile``: The file path of the ``.pem`` file that contains your
+  client certificate and private key.
+- ``authMechanism``: Set to ``'MONGODB-X509'``.
+
+You can set these options in two ways: by passing an options array to the
+``MongoDB\Client`` constructor or through parameters in your connection URI.
+
+.. tabs::
+
+   .. tab:: MongoDB\\Client
+      :tabid: Client
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-mongodb-X509-client
+         :end-before: end-mongodb-X509-client
+
+   .. tab:: Connection URI
+      :tabid: connectionstring
+
+      .. literalinclude:: /includes/authentication.php
+         :language: php
+         :dedent:
+         :start-after: start-mongodb-X509-uri
+         :end-before: end-mongodb-X509-uri
+
+.. _php-mongodb-aws:
+
+MONGODB-AWS
+-----------
+
+.. important::
+
+   The MONGODB-AWS authentication mechanism requires {+mdb-server+} v4.4 or later.
+
+The MONGODB-AWS authentication mechanism uses AWS IAM (Amazon Web Services Identity and
+Access Management) or AWS Lambda credentials to authenticate your application. To use this
+method to authenticate, you must specify ``'MONGODB-AWS'`` as the value of
+the ``authMechanism`` connection option.
+
+.. note::
+
+   The {+php-library+} uses the C driver's implementation of the MONGODB-AWS
```

Comment:
Optional suggestion: We can refer to the C driver as "libmongoc". That may help readers since we talk about libmongoc and libmongocrypt as dependencies elsewhere in our docs, and then "driver" will only refer to PHP in these tutorials.